packages/beacon-node/src/network/reqresp/handlers/lightClientUpdate.ts

```diff
@@ -0,0 +1,17 @@
+import {altair} from "@lodestar/types";
+import {MAX_REQUEST_LIGHT_CLIENT_UPDATES} from "@lodestar/params";
+import {IBeaconChain} from "../../../chain/index.js";
+import {ResponseError} from "../response/index.js";
+import {RespStatus} from "../../../constants/index.js";
+
+export async function* onLightclientUpdate(
+  requestBody: altair.LightClientUpdatesByRange,
+  chain: IBeaconChain
+): AsyncIterable<altair.LightClientUpdate[]> {
+  try {
+    const count = Math.min(MAX_REQUEST_LIGHT_CLIENT_UPDATES, requestBody.count);
+    yield await chain.lightClientServer.getUpdates(requestBody.startPeriod, count);
```

Comment:
this should be a `yield*` so each update gets yielded individually, rather than an array of updates being yielded.